Tradition and love

Cristiansarmiento123 opened the pull request:
se agrego request a dev

santiagovlskz committed:
Agregar sedes

santiagovlskz committed:
actualizacion de codigo

Cristiansarmiento123 committed:
Se hizo la seccion tradition con animaciones

Cristiansarmiento123 committed:
rama en dev con seccion tradition

mafe1226 commented:
Buenas tardes cristian, disculpa la tardanza estaba algo ocupada, espero estés bien, a continuación te comento los cambios que debes hacer:
HTML
1. Te falta ponerle al section la clase section que tenemos en los valores genéricas. 
2. en el div  que tienes la clase tradition__contenido te hace falta poner la clase container que tenemos en las variables genericas.
3. En la linea de codigo 6 es un div con clase"tradition__text", es mejor crear una lista y dentro de esta incluir las etiquetas de parrafo (p) y el read more.
4. El read more es un botón, asi que es mejor utilizar la etiqueta botton.
5. Los divs con clase numbers__card  es mejor una lista y dentro de esta incluir la img el h2 y el p. ( a slack te envio como puedes hacerlo).
Por el momento cambia esto y cuando vuelvas a hacer push y enviar los cambios vuelvo a mirar.
CSS
1.Donde tienes &__content no es necesario poner un width de 1200px ya que este ya se encuentra en container de variables genéricas lo mismo con el margin de ahí, cuando agregues esas clases elimina esto, igual donde tienes .numbers width: 1200px.
Eso seria todo por el momento, creo que no has llamado los cambios porque te deberian aparecer una carpeta utils con varibles (colores) y otra que dice commons y ahi estan las cosas  genericas y en partials está header. Para llamar esos cambios vas a la terminal o donde usualmente hagas y pones:
Tienes que poner git ckeckout dev y después git pull original dev y trae los cambios

Cristiansarmiento123 committed:
se agregaron clases

Cristiansarmiento123 committed:
se realizaron modifiaciones

Cristiansarmiento123 committed:
se actulizaron textos con listas

jairocaracas committed:
Responsive commit

-Se ajustaron un par de detalles en margenes y tamaños para que correspondiera el proyecto con el diseño en figma

- Se realizo el diseño responsivo para esta sección

jairocaracas committed:
Subtitle padding adjusted

Se ajusto el padding del subtitulo para que correspondiera con el diseño
